web/src/components/DefiningMutations/SelectReferenceDropdown.tsx

```diff
@@ -33,11 +37,20 @@ export function SelectReferenceDropdown({
       </label>
       <Dropdown
         identifier={id}
+        className={'width-5xs'}
         options={options}
-        value={stringToOption(selectedSequence)}
+        value={stringToOptionWithLabel(selectedSequence, toSentenceCase)}
         onOptionChange={onChange}
         isSearchable={true}
       />
     </div>
   )
 }
+
+const toSentenceCase = (str: string) => {
+  const s = str?.match(/[A-Z]{2,}(?=[A-Z][a-z]+[0-9]*|\b)|[A-Z]?[a-z]+[0-9]*|[A-Z]|[0-9]+/g)?.join(' ')
+  if (s === undefined) {
+    return str
+  }
+  return s.slice(0, 1).toUpperCase() + s.slice(1)
```

Comment:
```suggestion
return _.capitalize(_.startCase(str))
```
I think this does the same and is easier to follow.